packages/itwinui-css/src/listbox/listbox.scss

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+
+@mixin iui-listbox-wrapper {
+  --iui-menu-listbox-height: calc((var(--iui-component-height) - 1px) * 4.5);
+
+  display: flex;
+  gap: var(--iui-size-xs);
+
+  > :where(:first-child, :last-child) {
+    flex: 1;
```

Comment:
`:first-child`/`:last-child` selectors are not very reliable. maybe `grid` will be better as it will let you bypass the need for these.

Reply:
Yeah, `grid` might be a better choice here.  Using `grid` now.